packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -276,21 +229,31 @@ const props = defineProps<{
 const emit = defineEmits(['update-project']);
 
 const tabStore = useTabStore();
-const router = useRouter();
-const resources = useResourcesStore();
 
-const newModelId = ref<string>('');
-const isNewModel = ref<boolean>(true);
+// Makes asset/process from workflow node appear
+const openedWorkflowNodeStore = useOpenedWorkflowNodeStore();
+
+// Grabs the model id from the last model config output from the node
+const workflowNodeAssetId = computed(() => {
+	let index: number = 0;
+
+	if (openedWorkflowNodeStore?.workflowNode?.outputs.length) {
+		index = openedWorkflowNodeStore.workflowNode.outputs.length - 1; // Grab index of last output port
+	}
+	return (
+		openedWorkflowNodeStore?.workflowNode?.outputs[index].value.model.id.toString() ?? undefined
```

Comment:
Thinking about how to make this more generic to support the other drilldowns... maybe `openedWorkflowNodeStore` should just store a `assetType` and `assetId`?